Week1_final.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Maybe a note above the code saying what the code does and letting the student know they don't have to worry about the code, would be helpful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>